index.js

```diff
@@ -0,0 +1,189 @@
+function toggleDropdownMenu(arrowId) {
+  var arrow = document.getElementById(arrowId);
+
+  if (arrow.classList.contains("bi-caret-down-fill")) {
+    arrow.classList.remove("bi-caret-down-fill");
+    arrow.classList.add("bi-caret-up-fill");
+  } else {
+    arrow.classList.remove("bi-caret-up-fill");
+    arrow.classList.add("bi-caret-down-fill");
+  }
+}
+
+function filterFood() {
+  const priceRange = uislider.noUiSlider.get();
+  const minPrice = parseFloat(priceRange[0]);
+  const maxPrice = parseFloat(priceRange[1]);
+
+  const foodItems = document.getElementsByClassName("food-item");
+
+  Array.from(foodItems).forEach((foodItem) => {
+    const foodPrice = parseFloat(foodItem.dataset.price);
+
+    if (foodPrice >= minPrice && foodPrice <= maxPrice) {
+      foodItem.style.display = "block";
+    } else {
+      foodItem.style.display = "none";
+    }
+  });
+}
+
+function toggleCart() {
+  if (document.getElementById("cart-container").style.display == "none") {
+    document.getElementById("cart-container").style.display = "block";
+  } else {
+    document.getElementById("cart-container").style.display = "none";
+  }
+}
+
+document.addEventListener("DOMContentLoaded", (event) => {
+  let foodData = {
+    foods: [
```

Comment:
I like that you chose this approach for having the food data into a hash. To improve it, you could also export just this data into another file and import it here.

This way, if you need the same data in 2 places, you can have it from 1 source and also, the file would look smaller & easier to read.

It could look something like this:
```
# in a new file, foods.js
let foodData = {
    foods: [
        { name: 'Food 1', description: 'Type 1', price: 100 },
        { name: 'Food 2', description: 'Type 2', price: 102 },
        // Add more food objects here...
    ]
};

export default foodData;
```

and back here you can use
```
import foodData from './foodData.js';

# use directly foodData
```